### PR TITLE
🔎 Improve googlability - add Blazor to title

### PR DESCRIPTION
**Pull request by bradystroud**

### CODE REVIEW

File: rules/share-common-types-and-logic/rule.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 type: rule
-title: Do you share common types and logic?
+title: Blazor - Do you share common types and logic?
```
Comment (jaydenalchin) [inline rules/share-common-types-and-logic/rule.md:3]: Bad content design - change to "Do you share common types and logic in Blazor?"

**Review by jaydenalchin — CHANGES_REQUESTED** [review]

File: rules/share-common-types-and-logic/rule.md (hunk 1 above)
Comment (adamcogan) [reply rules/share-common-types-and-logic/rule.md:3]: Adding @pierssinclairssw 

Jayden - I disagree as I generally like prefixes.
See https://www.ssw.com.au/rules/prefixes/
-a

**Review by adamcogan — APPROVED** [review]

File: rules/share-common-types-and-logic/rule.md (hunk 1 above)
Comment (jaydenalchin) [reply rules/share-common-types-and-logic/rule.md:3]: @adamcogan  ⚠️ For the record I've downvoted that prefixes rule, please see the email thread "Re: Clean Architecture Rules - Add prefix" for some specific feedback from Tiago and I on this topic.

**Review by tiagov8 — APPROVED** [review]

**Review by tiagov8 — APPROVED** [review]

**Comment by tiagov8** [comment]
Hey @jaydenalchin 

I agree with you and overwrite this one
1. Please add your review

**Review by jaydenalchin — APPROVED** [review]

**Pull request merged**